tests/generated.go

```diff
@@ -683,34 +663,21 @@ func parsePairStorageNew(opts []Pair) (pairStorageNew, error) {
 			result.WorkDir = v.Value.(string)
 		// Default pairs
 		case "default_content_type":
-			defaultPairs = true
+			result.HasDefaultStoragePairs = true
 			result.DefaultStoragePairs.Create = append(result.DefaultStoragePairs.Create, pairs.WithContentType(v.Value.(string)))
 			result.DefaultStoragePairs.Write = append(result.DefaultStoragePairs.Write, pairs.WithContentType(v.Value.(string)))
 		case "default_storage_class":
-			defaultPairs = true
+			result.HasDefaultStoragePairs = true
 			result.DefaultStoragePairs.Write = append(result.DefaultStoragePairs.Write, WithStorageClass(v.Value.(string)))
 		// Enable features
 		case "enable_loose_pair":
-			if result.HasStorageFeatures {
-				continue
-			}
-			enableFeatures = true
+			result.HasStorageFeatures = true
 			result.StorageFeatures.LoosePair = v.Value.(bool)
```

Comment:
I found the last one will be picked if there are multiple `enable_loose_pair`.

Reply:
How about generating `HasEnableLoose` to prevent it?

Maybe treat `enable_loose` as a normal pair can prevent those strange edge cases?